### ISSUE TITLE
Segmentation fault for "ip addr show"

### ISSUE THREAD

**Issue by arthurbarr** [rancherio/os#44]
Running the command `ip addr show` results in a segmentation fault.  Running `ifconfig` works OK.

~~~
[rancher@rancher ~]$ ip addr show
1: lo: <LOOPBACK> mtu 65536 qdisc noop
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
2: eth0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc pfifo_fast qlen 1000
    link/ether 08:00:27:fc:5f:eb brd ff:ff:ff:ff:ff:ff
    inet 10.0.2.15/24 brd 10.0.2.255 scope global eth0
       valid_lft forever preferred_lft forever
Segmentation fault
~~~

**Comment by ibuildthecloud** [comment]
This is all busybox stuff.  I've noticed this one too.  If you run dmesg or look at /var/log/messages it will show the actual segfault address.  Since we strip the binaries of all debug symbols it's difficult to debug.  I think I need to produce the debug symbols to we can more easily troubleshoot issue like this.

**Comment by cloudnautique** [comment]
This issue has been addressed in rancherio/os-base#4 should be coming to the OS soon.

**Comment by arthurbarr** [comment]
Verified in V0.2.1 as fixed.
